src/cascade/input_data/configuration/builder.py

```diff
@@ -56,6 +57,8 @@ def initial_context_from_epiviz(configuration):
     context.parameters.rate_case = configuration.model.rate_case
     context.parameters.minimum_meas_cv = configuration.model.minimum_meas_cv
 
+    context.policies = policies_from_settings(configuration)
```

Comment:
Ah! I was concerned that you need settings to have policies. That would mean that you need the IHME part in order to run the non-IHME code, which would be bad. This step here is enough to separate the two.